src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctype/InputJavadocTypeUnusedParamInJavadocForClass.java

```diff
@@ -9,3 +9,4 @@
  */
 public class InputJavadocTypeUnusedParamInJavadocForClass {
 }
+
```

Comment:
remove unnecessary extra line.

Reply:
Comments about blank lines and formatting (your next comment), is exactly the value that checkstyle brings. You shouldn't spend your energy on these things, checkstyle should catch them for you. I find it a bit ironic to see this type of comments on a PR on checkstyle itself. You can see how I am using the EmptyLineSeparator rule here https://github.com/ngeor/checkstyle-rules to ensure vertical space inconsistencies break the build (and are not subject to code review).

Reply:
Yes I agree, but as checkstyle doesn't catch them currently, I have to keep pointing them out. We don't really use alot of checks on inputs which is why there is no violation.
Besides this new line was the only change to the file which was odd by itself.

Reply:
May I suggest to create an issue in the project so that someone eventually will implement this check?

Reply:
@ngeor ,
please create issue.

Reply:
Created issue #6110